core-bundle/src/Resources/contao/modules/Module.php

```diff
@@ -313,6 +313,13 @@ protected function renderNavigation($pid, $level=1, $host=null, $language=null)
 				trigger_deprecation('contao/core-bundle', '4.12', 'Using a tabindex value greater than 0 has been deprecated and will no longer work in Contao 5.0.');
 			}
 
+			// Hide the page if it is not protected and only visible to guests (backwards compatibility)
+			if ($objSubpage->guests && !$objSubpage->protected && $user)
```

Comment:
there's no `$user`?

Reply:
Line 281 and 285?

Reply:
not sure what you're looking at, but this has been removed by my frontend group voter and probably a rebase

<img width="883" alt="Bildschirmfoto 2021-07-05 um 08 32 32" src="https://user-images.githubusercontent.com/1073273/124427403-a2f2ea80-dd6b-11eb-86e5-7d3a07def1f3.png">

Reply:
You are right, I looked at the file before the rebase/merge https://github.com/contao/contao/blob/79a8dcff1c6805b15ff3820cf4f6c6c854a83ddd/core-bundle/src/Resources/contao/modules/Module.php#L281-L285

So this needs to be fixed indeed.

Reply:
Fixed in d3d00c9e709d7de8084d293f16f2ca25f247e1fd. Is it correct this way or did I have to check `ROLE_MEMBER` instead?

Reply:
never check the user, always ask a voter (= check `ROLE_MEMBER`)